### ISSUE TITLE
Crossbar problem!

### ISSUE THREAD

**Issue by woogenius** [Goluis/cottz-publish-relations#6]
Hi Goluis

I just started using this package. And I got a lot of code reduce. Thank for making this great package.
Now I'm trying to use Crossbar API and I got error now

```
I20160531-03:12:50.404(9)? Exception while invoking method 'PR.fireListener' ReferenceError: Crossbar is not defined
I20160531-03:12:50.405(9)?     at [object Object].PRFireListener (packages/cottz:publish-relations/lib/server/methods.js:17:3)
I20160531-03:12:50.405(9)?     at maybeAuditArgumentChecks (packages/ddp-server/livedata_server.js:1704:12)
I20160531-03:12:50.405(9)?     at packages/ddp-server/livedata_server.js:711:19
I20160531-03:12:50.406(9)?     at [object Object]._.extend.withValue (packages/meteor/dynamics_nodejs.js:56:1)
I20160531-03:12:50.406(9)?     at packages/ddp-server/livedata_server.js:709:40
I20160531-03:12:50.406(9)?     at [object Object]._.extend.withValue (packages/meteor/dynamics_nodejs.js:56:1)
I20160531-03:12:50.406(9)?     at packages/ddp-server/livedata_server.js:707:46
I20160531-03:12:50.406(9)?     at tryCallTwo (/Users/woogenius/.meteor/packages/promise/.0.6.7.22t7ot++os+web.browser+web.cordova/npm/node_modules/meteor-promise/node_modules/promise/lib/core.js:45:5)
I20160531-03:12:50.406(9)?     at doResolve (/Users/woogenius/.meteor/packages/promise/.0.6.7.22t7ot++os+web.browser+web.cordova/npm/node_modules/meteor-promise/node_modules/promise/lib/core.js:200:13)
I20160531-03:12:50.406(9)?     at new Promise (/Users/woogenius/.meteor/packages/promise/.0.6.7.22t7ot++os+web.browser+web.cordova/npm/node_modules/meteor-promise/node_modules/promise/lib/core.js:66:3)
```

I think there's a typo on your package. I'll try to fix and make PR

WooJin.

**Comment by lfades** [comment]
sorry about the mistake, I had to quickly pass the package to Meteor 1.3 to also pass my application to Meteor 1.3, and did not have time to properly test it. This week I am in partial, when finished I'll be able to fix the error

**Comment by woogenius** [comment]
@Goluis Thanks for your comment.

FYI, I tried bunch of things to make crossbar API working. but failed.

At first, I added this 2 lines on `lib/server/methods.js` and that error was gone.

```
import { DDPServer } from 'meteor/ddp-server';
const Crossbar = DDPServer._InvalidationCrossbar;
```

Then, I tried to excute sample code on `README.md`(listen - fire code) and I have never been succeeded.
Also, I tried to install 2.0.3 version of this package. but failed.

the situation was listen function block is called, and payload data is changed properly, then `this.cursor` function is called. But Client couldn't get any change of data from DDP.

![image](https://cloud.githubusercontent.com/assets/3888608/15657663/d19b407e-26ed-11e6-93b6-08a34eaea651.png)

Is there any particular thing that I need to know to use crossbar API?

**Comment by lfades** [comment]
Hi, I'm done with my partials and check the package.

apparently I forgot to pass `methods.js` correctly to meteor 1.3 (I lacked all imports and crossbar was not defined)

the above is already fixed, other changes I made were as follows:
- I no longer assign `this` to the `callback` in `this.listen` in favor of arrow functions.
- tabs to two spaces in all files.

Sorry for the inconvenience, it was my fault.
